### ISSUE TITLE
Is there a way to set the font for CJK characters separately?

### ISSUE THREAD

**Issue by nemo-nullius** [Tetralet/LilyTerm#123]
At first I want to say I like LilyTerm very much. It is one of the best Terminal Emulators I have ever used for it is quick, pretty, and can display CJK characters very well.

I am just wondering whether I can set the font for English alphabets and CJK characters separately, just like the options in .vimrc `guifont` and `guifontwide` for gvim. As I spend most of my time each day writing things in vim in the terminal, it would help me a lot if I could use a different font for CJK characters rather than the few ones originally designed for English alphabets.

Thanks for your help!

**Comment by Tetralet** [comment]
We may try to "bind" an English font and a CJK font into a single font by editing ~/.config/fontconfig/fonts.conf:

<match target="pattern" >

    <test compare="eq" name="family" >

        <string>monospace</string>

    </test>

    <edit mode="assign" binding="strong" name="family" >

        <string>DejaVu Sans Mono</string>

        <string>WenQuanYi Zen Hei</string>

    </edit>

</match>

http://lilyterm.luna.com.tw/html/document.html#Monospace%20fonts

**Comment by nemo-nullius** [comment]
All right... Now I see what the problem is. I have to restart the X in order for fonts.conf to take effect in LilyTerm. How stupid I am!

However, the configuration provided in the link above does not seem to be a good fit for my system though. The problem that I showed above still remains. After trying many times, I happened to write the following codes which fit me most in my case:

```
     <match target="pattern">
         <test qual="any" name="family" compare="eq">
             <string>Ubuntu Mono derivative Powerline</string>
         </test>
         <edit name="family" mode="append" binding="strong">                                                        
             <string>WenQuanYi Zen Hei</string>
         </edit>
     </match>
```

Ubuntu Mono derivative Powerline is for English alphabets as well as WenQuanYi Zen Hei for CJK characters, and both are arranged in order now:

![good](https://user-images.githubusercontent.com/17632396/30733211-a5e7f5be-9fa8-11e7-8ac3-9e06db5df3a0.png)

But there are still some questions perplexing me all the time. For I have also set a font in LilyTerm, what's the relationship between this font and the font I set in fonts.conf? What's more, what are the functions of such elements and expressions, like `mode=append`, `name="family"``, `qual="any"`, in the configuration file? Although I have checked the man page of fonts.conf, I still could not fully get it. Is there any document on this in detail?

I really appreciate it a lot if you could help me!

**Comment by Tetralet** [comment]
How about using the Noto fonts?

https://github.com/adobe-fonts/source-han-sans/tree/release

**Comment by nemo-nullius** [comment]
Yes, I am also using that, including both Souce Code Pro and Source Han Sans. They are really pretty fonts.  : ) But I still need to change `mode="assign"` to `mode="append"` in `fonts.conf` in order to make both fonts display correctly.

Here is my setting:
```

     <match target="pattern">
          <test qual="any" name="family" compare="eq">
              <string>Source Code Pro for Powerline</string>
          </test>
          <edit name="family" mode="append" binding="strong">
              <string>Source Han Sans Regular</string>
          </edit>
      </match>
```
